engine/src/flutter/impeller/display_list/dl_dispatcher.cc

```diff
@@ -598,6 +598,7 @@ void DlDispatcherBase::drawDiffRoundRect(const DlRoundRect& outer,
   PathBuilder builder;
   builder.AddRoundRect(outer);
   builder.AddRoundRect(inner);
+  builder.SetBounds(outer.GetBounds().Union(inner.GetBounds()));
```

Comment:
While we're at it, the original issue shows a lot of overhead from us re-computing this value, but we don't need to.